db/data/dev/001_intake.sql

```diff
@@ -31,4 +31,6 @@ open_timestamp = excluded.open_timestamp,
 close_timestamp = excluded.close_timestamp,
 ccbc_intake_number = excluded.ccbc_intake_number;
 
+select setval('ccbc_public.intake_id_seq', 2, true);
```

Comment:
Hey I appreciate this, I just tried it but looks like we also need to set the gapless counter id sequence. I'll approve this but it needs this small update 🙇‍♂️ 

```suggestion
select setval('ccbc_public.intake_id_seq', 2, true);
select setval('ccbc_public.gapless_counter_id_seq', 2, true);
```

Reply:
Done!